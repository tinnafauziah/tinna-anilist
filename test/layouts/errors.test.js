import errorLayout from '~/layouts/error';
describe('Layout Error', () => {
  const mockPage = () => {
    const propsData = {};
    return global.shallowMount(errorLayout, { propsData });
  };
  it('should layout page match snapshot', () => {
    const wrapper = mockPage();
    expect(wrapper.vm.error);
    expect(wrapper.element).toMatchSnapshot();
  });
});
