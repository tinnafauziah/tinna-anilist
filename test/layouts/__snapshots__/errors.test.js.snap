// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Error should layout page match snapshot 1`] = `
<v-app-stub
  id="app"
>
  <div
    class="display-1"
  >
    An error occurred
  </div>
  <div
    class="subtitle-1"
  >
    Please return to
    <span>
      Â 
      <a>
        Home page
      </a>
    </span>
  </div>
</v-app-stub>
`;
